app/src/main/java/com/duycuong/weather/utils/ConvertTime.java

```diff
@@ -0,0 +1,17 @@
+package com.duycuong.weather.utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Created by DuyCương on 08/02/2018.
+ */
+
+public class ConvertTime {
+    public static String getCurrentTime()
+    {
+        Date date= new Date();
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("HH:mm dd/MM/yyyy");
```

Comment:
WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.